### ISSUE TITLE
Access to the all recognition lifecycle events on Javascript SDK 1.0.1.

### ISSUE THREAD

**Issue by npacucci** [Azure-Samples/cognitive-services-speech-sdk#85]
Hi I want to know if it's possible to access to the all recognition lifecycle events (_RecognitionTriggeredEvent, ListeningStartedEvent, RecognitionStartedEvent, etc._) as the previous **[SpeechToText-WebSockets-Javascript](https://github.com/Azure-Samples/SpeechToText-WebSockets-Javascript)** **RecognizerStart()** function.

I'm using the [Javascript SDK 1.0.1](https://docs.microsoft.com/it-it/azure/cognitive-services/speech-service/quickstart-js-browser) version and I tried the Sample implementation by using the **recognizeOnceAsync()** function but it returns the result directly when the recognition ends.

Thanks!

**Comment by rhurey** [comment]
When using recognizeOnceAsync() the recognizer will fire the related events, so you can get results from either the continuation method or the events.

You can see https://github.com/Azure-Samples/cognitive-services-speech-sdk/blob/master/samples/js/browser/index.html for an example of when events are fired during recognizeOnceAsync()

**Comment by npacucci** [comment]
Yes, it's important to setup the callbacks to be invoked when an event occurs, such as:

reco.recognizing = function (s, e) {
       // . . . on recognizing
}

This pattern is different from the previous Bing Speech SDK, but it also works fine!
